common/app/views/fragments/image.scala.html

```diff
@@ -0,0 +1,59 @@
+@import layout.ContentWidths.MainMedia
+@import views.support.ImgSrc
+
+@(
+    picture: model.ImageMedia,
+    isShowcase: Boolean = false,
+    isFeature: Boolean = false,
+    widths: layout.WidthsByBreakpoint =  MainMedia.inline,
```

Comment:
Galleries main media will currently be using the default `widths` specified here. Do we need to specify our own `widths`?